LocalStoreSimulation/src/ro/scit/menu/Menu.java

```diff
@@ -0,0 +1,146 @@
+package ro.scit.menu;
+
+import entities.Animal;
+import entities.Product;
+import entities.Vegetable;
+import ro.scit.storeentities.SoldItem;
+import ro.scit.storeentities.StockItem;
+
+import java.util.*;
+
+/**
+ * @author Georgiana
+ * Menu Class
+ */
+
+
+public class Menu {
+    Scanner sc = new Scanner(System.in);
+    List<Product> productList = new ArrayList<>();
+    List<SoldItem>  orderList = new ArrayList<>();
+    List <StockItem> stockList = new ArrayList<>();
+
+    Product product = new Product();
+    StockItem stock = new StockItem(new Product());
+
+    public void display() {
+        int selection = 0;
+        while (selection != 4) {
+            System.out.println(
+                    "Select an option: \n" +
+                            "  1) Create product and add it to stock\n" +
+                            "  2) Sell product\n" +
+                            "  3) Display daily sales report \n" +
+                            "  4) Exit\n "
+            );
+
+            selection = sc.nextInt();
+            sc.nextLine();
+            switch (selection) {
+                case 1:
+                    System.out.println("Product type (1 - animal; 2 - vegetable): ");
+                    int type = sc.nextInt();
+                    //Animal product
+                    if (type == 1) {
+                        int id = 0;
+                        Animal animal = new Animal();
+                        animal.setType("ANIMAL");
+                        System.out.println("Animal products: milk; eggs; cheese; yougurt; sourcream)");
+                        String animalName = sc.next();
+                        if (animalName.equals("milk")) {
+                            animal.getMILK();
```

Comment:
calling getMilk here will display a message and return MILK but that value is ignored